fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -70,7 +75,6 @@ public String getFactTableName() {
 
     @Override
     public String toString() {
-        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames() + " alignment: " +
-                getTableAlignment();
+        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames();
```

Comment:
Blows up because `getTableAlignment` calls `getAllAvailableIntervals` which will in turn go to `DatasourceMetadataservice` to look for a non-existing datasource for segment data which will throw an exception directly.